app-admin/fleet/files/sysusers.conf

```diff
@@ -0,0 +1,4 @@
+# create fleet group
+g fleet 253 - -
```

Comment:
253 is available, although since the ebuild doesn't set the ownership of anything or run tmpfiles in advance I think we can set it to `-` and let systemd allocate the id on first boot. Static ids should only be needed when maintaining compatibility with the old scheme in baselayout or the pre-built filesystems have ownership set for some reason.